docs/development/local-e2e-tests.md

```diff
@@ -33,12 +33,13 @@ The following providers are supported for e2e tests:
 
 An e2e test execution involves the following steps:
 
-| Step   	    | Description                                                                                                                                                                                                |
-|-------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `setup`	    | Create a storage bucket which is used for etcd backups (only with cloud providers). 	                                                                                                                      |
-| `deploy`	   | Build Docker image, upload it to registry (if remote cluster - see [Docker build](https://skaffold.dev/docs/pipeline-stages/builders/docker/)), deploy Helm chart (`charts/druid`) to Kubernetes cluster.	 |
-| `test`      | Execute e2e tests as defined in `test/e2e`.	                                                                                                                                                               |
-| `cleanup`   | Delete storage bucket and Druid deployment from test cluster.	                                                                                                                                             |
+| Step   	  | Description                                                                                                                                                                                                |
+|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `setup`	  | Create a storage bucket which is used for etcd backups (only with cloud providers). 	                                                                                                                      |
+| `deploy`	 | Build Docker image, upload it to registry (if remote cluster - see [Docker build](https://skaffold.dev/docs/pipeline-stages/builders/docker/)), deploy Helm chart (`charts/druid`) to Kubernetes cluster.	 |
+| `test`    | Execute e2e tests as defined in `test/e2e`.                                                                                                                                                                |
+| `undeploy` | Remove the deployed artifacts from Kubernetes cluster.                                                                                                                                                     |
+| `cleanup` | Delete storage bucket and Druid deployment from test cluster.	                                                                                                                                             |
```

Comment:
Isn't druid deployment get deleted in `undeploy` step ?

Reply:
Yes it is. The Druid `deployment` is one of the deployed artifacts 🤔

Reply:
ok, what I'm asking is `cleanup` also does that ? as it is mentioned `Delete storage bucket and Druid deployment from test cluster`